### PR TITLE
Heroku Deployment

### PR DESCRIPTION
**Pull request by stanleyye**
Make sure to test it out on heroku first. I also wrote a mini guide on deploying it. More details can be found in the links.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+#Deploying to Heroku
+
+First make sure you have Node.js (which comes with npm as well) installed. 
+Access to the heroku and gmail acc can be found in the Slack channel.
+
+Then install the Heroku CLI (command-line interface) which also comes with a Git installer. 
+
+The app dependencies have already been installed. It is under dependencies in the package.json file. 
+**The version of Node.js that we are currently using is 6.9.5 (subject to change).** It will be used to run the app on Heroku.
```
Comment (jeffries) [inline README.md:9]: Why 6.9.5?

**Comment by stanleyye** [comment]
Note: on Heroku, it drops listening after around 30s because it's not binded to a port.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+Start your app locally. You should be able to see the processes with your shell. The app should be listening on **localhost:3000**.
+
+```
+heroku local web
+```
+
+###Deploying to Heroku
+```
+git add -A 				
+git commit -m "commit message"
```
Comment (jeffries) [inline README.md:32]: Are these `git` commands necessary for a routine deployment?

File: README.md (hunk 1 above)
Comment (stanleyye) [reply README.md:9]: That's the recommended one when you install Node. That being said, odd ones aren't stable yet, we can switch to 6.9.4 or any other stable one

File: README.md (hunk 2 above)
Comment (stanleyye) [reply README.md:32]: I think so but do correct me if i'm wrong. Heroku heavily relies on Git

File: README.md (hunk 1 above)
Comment (jeffries) [reply README.md:9]: Recommended by Heroku?

File: README.md (hunk 2 above)
Comment (jeffries) [reply README.md:32]: I only think you should be doing those things if you're adding code to the repo; if you're just deploying these shouldn't be run

**Review by jeffries — CHANGES_REQUESTED** [review]
Looks good to me. Minor change to deployment instructions necessary.

File: README.md (hunk 1 above)
Comment (stanleyye) [reply README.md:9]: nope when you install Nodejs

File: README.md (hunk 2 above)
Comment (stanleyye) [reply README.md:32]: https://devcenter.heroku.com/articles/git

Heroku says : Before you can push an app to Heroku, you’ll need to initialize a local Git repository and commit your files to it

so I think it's necessary?

**Comment by stanleyye** [comment]
check my other pull req which I squashed some commits.

**Pull request closed without merging**